lib/controllers/sharks.js

```diff
@@ -0,0 +1,44 @@
+const { Router } = require('express');
+const { Shark } = require('../models/Shark');
+
+module.exports = Router()
+  .delete('/:id', async (req, res, next) => {
+    try {
+      const data = await Shark.delete(req.params.id);
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .post('/', async (req, res, next) => {
+    try {
+      const data = await Shark.insert(req.body);
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .put('/:id', async (req, res, next) => {
+    try {
+      const data = await Shark.updateById(req.params.id, req.body);
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .get('/:id', async (req, res, next) => {
+    try {
+      const data = await Shark.getById(req.params.id, req.body);
+      res.json(data);
+    } catch (e) {
+      next(e);
+    }
+  })
```

Comment:
The response here sends a 200 when hitting a shark that is not present. Perhaps try removing the req.body from the getById argument as we only need to send the id to our model.